docs/source/tutorial/tutorial-subscriptions.md

```diff
@@ -0,0 +1,198 @@
+
+```swift:title=LaunchesViewController.swift
+// MARK: - Subscriptions
+
+private func startSubscription() {
+          activeSubscription = Network.shared.apollo.subscribe(subscription: TripsBookedSubscription()) { result in
+            switch result {
+            case .failure(let error):
+                self.showAlert(title: "NetworkError",
+                               message: error.localizedDescription)
+            case .success(let graphQLResult):
+                if let errors = graphQLResult.errors {
+                    self.showAlertForErrors(errors)
+                } else if let tripsBooked = graphQLResult.data?.tripsBooked {
+                    self.handleTripsBooked(value: tripsBooked)
+                } else {
+                    // There was no data and there were no errors, do nothing.
+                }
+            }
+        }
+    }  
+}
+
+private func handleSubscriptionEvent() {
+   print("Trips booked: \(value)")
+}
+```
+
+Finally, add a line to `viewDidLoad` which actually starts the subscription: 
+
+```swift:title=LaunchesViewController.swift
+override func viewDidLoad() {
+    super.viewDidLoad()
+    self.startSubscription()
+    self.loadMoreLaunchesIfTheyExist()
+}
+```
+
+Build and run your app and go back to Sandbox Explorer, and select the tab where you set up the `BookTrip` mutation. Book a new trip while your app is open, you'll see a log print out: 
+
+```
+Trips booked: 1
+```
+
+Cancel that same trip, and you'll see another log: 
+
+```
+Trips booked: -1
+```
+
+Now, let's display that information in a view! Replace the `print` statement in `handleTripsBooked` with code to use the included `NotificationView` to show a brief alert at the bottom of the screen with information about a trip being booked or cancelled: 
+
+```swift:title=LaunchesViewController.swift
+private func handleTripsBooked(value: Int) {
+        var message: String
+        switch value {
+        case 1:
+            message = "A new trip was booked! 🚀"
+        case -1:
+            message = "A trip was cancelled! 😭"
+        default:
+            self.showAlert(title: "Unexpected value",
+                           message: " Subscription returned unexpected value: \(value)")
+            return
+        }
+        
+        NotificationView.show(in: self.navigationController!.view,
+                              with: message,
+                              for: 4.0)
+    }
+```
+
+Build and run the application to your simulator, then use Studio to send bookings and cancellations again, and your iOS app should see some shiny new notifications pop up: 
+
+<img alt="A new trip was booked (rocket)" class="screenshot" src="images/screenshot_trip_booked.png" width="300"/>
+
+And you've done it! You've completed the tutorial. 
+
+## More resources
+
+There are way more things you can do with the Apollo iOS SDK, and the rest of this documentation includes info on more advanced topics like:
+
+- Using [fragments](/fragments/)
```

Comment:
These links 404 but I assume that's because you need to view this doc in the actual tutorial environment for the paths to route correctly?

Reply:
I think so? My recollection is that if it were to be something where it'd actually 404 that would fail the build